index.html

```diff
@@ -6835,8 +6905,11 @@ <h3 id="Mortality">Mortality</h3>
                </td>
                <td>Well of Renewal
                </td>
-               <td>Choose a point within 100ft, heal all creatures within 5ft for Medium+SPI
-                  (excludes self).
+               <td>
+                  <strong>Create a burst of healing energy that restores all nearby.</strong>
```

Comment:
probably should be all creatures, reads a bit janky

Reply:
Trying to generally avoid using the word Creatures in the flavour text because it's used as technical language, so I'd rather the two be mostly separate. "restores all in proximity"?